scripts/workflow/react-deploy.sh

```diff
@@ -3,6 +3,7 @@ export FRONT_END_DIR="${FRONT_END_DIR:-"src/frontend"}"
 export SPA_BUCKET_NAME="${SPA_BUCKET_NAME:-""}"
 export COMMIT_HASH="${COMMIT_HASH:-""}"
 export WORKSPACE="${WORKSPACE:-""}"
+export CUSTOM_ARTEFACT_LOCATION="${CUSTOM_ARTEFACT_LOCATION:-""}"
```

Comment:
Maybe one for later
I think but please check that WORKSPACE is only used to locate the react zip file in S3... the SPA_BUCKET_NAME identifies where the code is to be pushed
If so I think CUSTOM_ARTEFACT_LOCATION and the logic around its use may be redundant. Eg could we pass WORKSPACE and just use that - eg WORKSPACE is either eg dr-999 or default ?

Reply:
Ticket to be created

Reply:
DR-1029